docs/config.rst

```diff
@@ -1096,6 +1150,45 @@ Thresholds
 
    :attr:`LP_ENABLE_BATT` must be enabled to have any effect.
 
+.. attribute:: LP_DISK_PRECISION
+   :type: int
+   :value: 2
+
+   Control the numbers of decimals when displaying the absolute available space
+   of the current hard drive. If set to 0, don't display decimals. If set to 1
+   or 2, display decimals.
+
+   See :attr:`LP_ENABLE_DISK`, :attr:`LP_ALWAYS_DISPLAY_VALUES`, and
+   :attr:`LP_DISPLAY_VALUES_AS_PERCENTS`.
+
+   .. versionadded:: 2.2
+
+.. attribute:: LP_DISK_THRESHOLD
+   :type: int
+   :value: 100000
+
+   Display something if the available space on the hard drive hosting the
+   current directory goes below this absolute threshold *in kilobytes*.
+
+   The threshold for disk can also be set with :attr:`LP_DISK_THRESHOLD_PERC`.
```

Comment:
Make it more clear how these two config options interact. Can I only set one at a time? If both are set, is it an OR logic, where if either one matches, or AND?